.github/workflows/wp-scripts-lint.yml

```diff
@@ -67,6 +76,9 @@ jobs:
     env:
       NODE_OPTIONS: ${{ inputs.NODE_OPTIONS }}
       NODE_AUTH_TOKEN: ${{ secrets.NPM_REGISTRY_TOKEN }}
+      GITHUB_USER_SSH_KEY: ${{ secrets.GITHUB_USER_SSH_KEY }}
+      GITHUB_USER_NAME: ${{ secrets.GITHUB_USER_NAME }}
+      GITHUB_USER_EMAIL: ${{ secrets.GITHUB_USER_EMAIL }}
```

Comment:
Why do we set these environment variables here if we read the values from the _secrets_ later and only use the env vars to check for existence (which is unnecessary in one of two cases)?
```suggestion
```
If you agree with this change, please also update accordingly in https://github.com/inpsyde/reusable-workflows/blob/main/.github/workflows/build-and-push-assets.yml. Thanks!